src/angular-cli.json

```diff
@@ -42,7 +42,7 @@
     "prefixInterfaces": false,
     "styleExt": "css",
     "build": {
-      "showCircularDependencies": true
+      "showCircularDependencies": false

```

Comment:
dynamicComponents nesting (webserver component for example) introduced circular dependencies. We can't avoid it. Muting warning messages